SmartHouseGatewayApp/SmartHouseDataStore/Entities/Models/DeviceSetting.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
```

Comment:
Remove unnecessary namespace in all classes. I think VS 2019 can help you with that